README_new.md

```diff
@@ -0,0 +1,617 @@
+-   **-logAppend**. If used, the log lines are appended to the
+    contextBroker log file, instead of re-creating it when the
+    broker starts.
+-   **--silent**. Suppress all log output except errors.    
+-   **-t <trace level>**. Specifies the initial trace levels
+    for logging. You can use a single value (e.g. "-t 70"), a
+    range (e.g. "-t 20-80"), a comma-separated list (e.g. "-t 70,90") or
+    a combination of them (e.g. "-t 60,80-90"). If you want to use all
+    trace levels for logging, use "-t 0-255". Note that trace levels can
+    be changed dynamically using the [management REST
+    interface](docs/admin/management_api.md). The detail of the
+    available tracelevels and its numbers can be found
+    [here](https://github.com/telefonicaid/fiware-orion/blob/develop/src/lib/logMsg/traceLevels.h)
+    (as a C struct).
+-   **-fg**. Runs broker in foreground (useful for debugging)
+-   **-localIp <ip>**. Specifies in which IP interface the broker
+    listens to. By default it listens to all the interfaces.
+-   **-pidpath <pid_file>**. Specifies the file to store the PID of the
+    broker process.
+-   **-httpTimeout <interval>**. Specifies the timeout in milliseconds
+    for forwards and notifications.
+-   **-corsOrigin <domain>**. Configures CORS allowed for GET requests,
+    specifing the allowed origin (use `__ALL` for `*`).
+-   **-reqMutextPolicy <all|none|write|read>**. Specifies the internal
+    mutext policy. Possible values are: "all" (which ensures that in a
+    given CB node as much as 1 request is being processed by the
+    internal logic module at the same time), "read" (which ensures that
+    in a given CB node as much as 1 read request is being processed by
+    the internal logic module at the same time, write request can
+    execute concurrently), "write" (which ensures that in a given CB
+    node as much as 1 write request is being processed by the internal
+    logic module at the same time, write request can
+    execute concurrently) and "none" (which allows all the requests
+    being executed concurrently). Default value is "all". For
+    Active-Active Orion configuration "none" is recommended.
+-   **-mutexTimeStat**. Include semaphore waiting time in the
+    "/statistics" information. It may have performance impact.
+
+### Checking status
+
+Only available if you are running Orion Context Broker [as a system
+service](#As_system_service).
+
+In order to check the status of the broker, use the following command
+with superuser privileges (using the root user or the sudo command):
+
+    /etc/init.d/contextBroker status
+
+If broker is running you will get:
+
+    Checking contextBroker...                         Running
+
+If broker is not running you will get:
+
+    Checking contextBroker...                         Service not running
+
+## API Overview
+
+In order to create an entity (Room1) with two attributes (temperature and pressure):
```

Comment:
I understand that the cookbook is part of the API Walkthorugh/Examples that, as far as I understand by https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines, is in a different section (just below this one).